From cd173b21c42456bde9765e7f6164e495b6c6a303 Mon Sep 17 00:00:00 2001
From: James Feist <james.feist@linux.intel.com>
Date: Wed, 3 Jun 2020 10:08:05 -0700
Subject: [PATCH 1/1] Temporarily disable EventService log support

This is causing bmcweb to crash when there is no
redfish log.

Change-Id: Id53e57f1ca140ee44f69fec46fd63bec18490c99
Signed-off-by: James Feist <james.feist@linux.intel.com>
---
 src/webserver_main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/webserver_main.cpp b/src/webserver_main.cpp
index ea32122..421cd1d 100644
--- a/src/webserver_main.cpp
+++ b/src/webserver_main.cpp
@@ -116,7 +116,7 @@ int main(int argc, char** argv)
 
     redfish::RedfishService redfish(app);
 
-#ifndef BMCWEB_ENABLE_REDFISH_DBUS_LOG_ENTRIES
+#if 0
     int rc = redfish::EventServiceManager::startEventLogMonitor(*io);
     if (rc)
     {
-- 
2.17.1

